packages/augur-ui/src/modules/modal/reporting.tsx

```diff
@@ -204,16 +209,29 @@ export default class ModalReporting extends Component<
       numOutcomes,
       marketType,
       description: '',
-      attoRepAmount: this.state.inputtedReportingStake.inputToAttoRep,
+      attoRepAmount: estimateGas ? '100000000000000000000' : this.state.inputtedReportingStake.inputToAttoRep,
```

Comment:
Why use value of 100000000000000000000?

Reply:
Meant to use a constant. Using 1 REP as the placeholder value for the estimateGas param, didn't see any difference in terms of gas usage based on this value so anything >0 would work for this estimation request.